src/utils/date.spec.ts

```diff
@@ -63,6 +73,23 @@ describe("dateFromISO", () => {
   });
 });
 
+describe("dateToISO", () => {
+  it("returns empty string from bad input", () => {
+    expect(dateToISO("" as any)).toEqual("");
+    expect(dateToISO("asdflkjasdhoui" as any)).toEqual("");
+  });
+  it("correctly returns string in simplified ISO format (YYYY-MM-DD)", () => {
+    const date = new Date(2011, 10, 29);
+    const expectedValue = "2011-11-29";
+    expect(dateToISO(date)).toEqual(expectedValue);
+  });
```

Comment:
I think this test is redundant since the one below tests the same base case plus the zero padding. We could probably remove it, what do you think?